### PR TITLE
MCKIN-23599 Automating ooyala xblock extraction process in PO file

### PR DESCRIPTION
**Pull request by wasifarbisoft**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+EXTRACTED_TEXT := $(EXTRACT_DIR)/text.po
+EXTRACTED_DJANGOJS := $(EXTRACT_DIR)/djangojs-partial.po
+EXTRACTED_TEXTJS := $(EXTRACT_DIR)/textjs.po
+
+
+help: ## display this help message
+	@echo "Please use \`make <target>' where <target> is one of"
+	@perl -nle'print $& if m{^[a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
+
+extract_translations: ## extract strings to be translated, outputting .po files
```
Comment (naeem91) [inline Makefile:16]: Please add instructions in readme file for extracting translations. Same as https://github.com/open-craft/xblock-chat#translation-i18n

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,9 @@
 -e .
 -e git://github.com/open-craft/xblock-mentoring.git@1a6e2092a361f0befff18d07607517d437046e3b#egg=xblock-mentoring
 -e git://github.com/nosedjango/nosedjango.git@ed7d7f9aa969252ff799ec159f828eaa8c1cbc5a#egg=nosedjango-dev
+
+xblock==1.2.9
+django-statici18n==1.8.2
+edx-i18n-tools==0.4.7
+django-appconf==1.0.3; python_version == '2.7'
+django-appconf==1.0.4; python_version > '2.7'
```
Comment (naeem91) [inline requirements.txt:9]: These requirements are not meant for default installation so should be part of a separate requirements file e.g; `requirements-dev` etc. Also `Django` will be needed

**Review by naeem91 — CHANGES_REQUESTED** [review]

**Comment by wasifarbisoft** [comment]
@naeem91 
Pull Request is ready for another round of review.

File: README.md [hunk 3]
```diff
@@ -107,6 +107,36 @@ $ pushd $VIRTUAL_ENV/src/xblock-mentoring/; pip install -r requirements.txt; pop
...
+- `detect_changed_source_translations`. Use [`i18n_tool` changed](https://github.com/edx/i18n-tools) to identify any updated translations.
+- `validate_translations`. Compile translations and check the source translations haven't changed.
+
+If you want to add a new language:
+  1. Add language to `ooyala_player/translations/config.yaml`
+  2. Make sure all tagged strings have been extracted:
+  ```bash
+  make extract_translations
+  ```
+  3. Clone `en` directory to `ooyala_player/translations/<lang_code>/` for example: `problem_builder/translations/fa_IR/`
```
Comment (naeem91) [inline README.md:133]: Change `problem_builder` to `ooyala_player/`

**Review by naeem91 — APPROVED** [review]
LGTM. Just a minor readme change.

**Review by msaqib52 — APPROVED** [review]

**Pull request merged**